### ISSUE TITLE
TreeSelect删除节点时控制台报错  print error in console when delete node

### ISSUE THREAD

**Issue by hughfenghen** [ant-design/ant-design#8734]
### Version
3.0.3

### Environment
mac: 10.13.1，chrome: 63

### Reproduction link
[https://codepen.io/anon/pen/opBbae?&editors=001 ](https://codepen.io/anon/pen/opBbae?&editors=001 )

### Steps to reproduce
1. Open link：https://codepen.io/anon/pen/opBbae?&editors=001  (Doc demo)  
2. Enter `"0"` and select `"Child Node3"`
3. Delete `"Child Node3"`

### What is expected?
`"Child Node3"`  is deleted

### What is actually happening?
`"Child Node3"` is not deleted  
error log：  
```log
Uncaught TypeError: Cannot read property 'length' of undefined
    at isPositionPrefix (unpkg.com/antd@3.0.3/dist/antd-with-locales.js:27825)
```

**Comment by hughfenghen** [comment]
@afc163 你好，能帮忙确认下是bug吗？

**Comment by valleykid** [comment]
Fixed it by antd@3.2.3.
